component/loki/write/write.go

```diff
@@ -5,9 +5,10 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/grafana/agent/component/common/loki/client"
```

Comment:
I'm not sure why all these were transferred into their own lines. Is it something from an IDE?

Reply:
My bad! fixed this now